pkg/system/phase2_creating.go

```diff
@@ -170,7 +170,6 @@ func (r *Reconciler) SetDesiredNooBaaDB() error {
 	if r.NooBaaDB.UID == "" {
 		for i := range r.NooBaaDB.Spec.VolumeClaimTemplates {
 			pvc := &r.NooBaaDB.Spec.VolumeClaimTemplates[i]
-			pvc.Namespace = r.NooBaaDB.Namespace
```

Comment:
I think setting the PVC namespace is full proofing even if r.own does not need it in this version of the SDK

Reply:
You are probably right, but as this is not mandatory in this operator-sdk version, we can keep it out for now - for the sake of having a complete revert of the PR.
the line is still in the master - So we will keep it for future releases